clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/UpsertGoogleInternalLoadBalancerAtomicOperation.groovy

```diff
@@ -241,30 +242,78 @@ class UpsertGoogleInternalLoadBalancerAtomicOperation extends GoogleAtomicOperat
         subnetwork: subnet.selfLink,
         ports: description.ports
       )
-      Operation forwardingRuleOp = safeRetry.doRetry(
-        { timeExecute(
-              compute.forwardingRules().insert(project, region, forwardingRule),
-              "compute.forwardingRules.insert",
+       insertRegionalForwardingRule(compute, project, region, forwardingRule)
+    } else if (existingForwardingRule && needToUpdateForwardingRule) {
+      def network = GCEUtil.queryNetwork(description.accountName, description.network, task, BASE_PHASE, googleNetworkProvider)
```

Comment:
The `network` and `subnet` are already fetched just inside the `if (!existingForwardingRule) {`.